src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -81,6 +83,7 @@ std::string getEntities
   std::string  geometry   = ciP->uriParam["geometry"];
   std::string  coords     = ciP->uriParam["coords"];
   std::string  out;
+  bool         isPattern  = true;
```

Comment:
Not sure why this is needed. Note that, no matter if id= or idPattern= is used, mongoBackend needs always patterns (note how the entity id is built, even in the case of a single id= element).